ext/intl/php_intl.stub.php

```diff
@@ -39,35 +39,35 @@ function intlcal_roll(IntlCalendar $calendar, int $field, $value): bool {}
 
 function intlcal_clear(IntlCalendar $calendar, ?int $field = null): bool {}
 
-function intlcal_field_difference(IntlCalendar $calendar, float $timestamp, int $field): int {}
+function intlcal_field_difference(IntlCalendar $calendar, float $timestamp, int $field): int|false {}
 
-function intlcal_get_actual_maximum(IntlCalendar $calendar, int $field): int {}
+function intlcal_get_actual_maximum(IntlCalendar $calendar, int $field): int|false {}
 
-function intlcal_get_actual_minimum(IntlCalendar $calendar, int $field): int {}
+function intlcal_get_actual_minimum(IntlCalendar $calendar, int $field): int|false {}
 
-function intlcal_get_day_of_week_type(IntlCalendar $calendar, int $dayOfWeek): int {}
+function intlcal_get_day_of_week_type(IntlCalendar $calendar, int $dayOfWeek): int|false {}
 
-function intlcal_get_first_day_of_week(IntlCalendar $calendar): int {}
+function intlcal_get_first_day_of_week(IntlCalendar $calendar): int|false {}
 
-function intlcal_get_least_maximum(IntlCalendar $calendar, int $field): int {}
+function intlcal_get_least_maximum(IntlCalendar $calendar, int $field): int|false {}
 
-function intlcal_get_greatest_minimum(IntlCalendar $calendar, int $field): int {}
+function intlcal_get_greatest_minimum(IntlCalendar $calendar, int $field): int|false {}
 
-function intlcal_get_locale(IntlCalendar $calendar, int $type): string {}
+function intlcal_get_locale(IntlCalendar $calendar, int $type): string|false {}
 
-function intlcal_get_maximum(IntlCalendar $calendar, int $field): int {}
+function intlcal_get_maximum(IntlCalendar $calendar, int $field): int|false {}
 
-function intlcal_get_minimal_days_in_first_week(IntlCalendar $calendar): int {}
+function intlcal_get_minimal_days_in_first_week(IntlCalendar $calendar): int|false {}
 
 function intlcal_set_minimal_days_in_first_week(IntlCalendar $calendar, int $days): bool {}
 
-function intlcal_get_minimum(IntlCalendar $calendar, int $field): int {}
+function intlcal_get_minimum(IntlCalendar $calendar, int $field): int|false {}
 
 function intlcal_get_time_zone(IntlCalendar $calendar): IntlTimeZone|false {}
 
 function intlcal_get_type(IntlCalendar $calendar): string {}
 
-function intlcal_get_weekend_transition(IntlCalendar $calendar, int $dayOfWeek): int {}
+function intlcal_get_weekend_transition(IntlCalendar $calendar, int $dayOfWeek): int|false {}
```

Comment:
We should backport these changes to 8.0, right?

Reply:
Yes.. :( Let me do that soon

Reply:
Went ahead and did this in https://github.com/php/php-src/commit/2afbacc16df1ea9b1d804ff6a8d34c07924c7968.

Reply:
Thanks!